run_studio.py

```diff
@@ -2,20 +2,22 @@
 
 import sys
 
-if sys.version_info < (3, 7, 0):  # pragma: no cover
+if sys.version_info < (3, 8, 0):  # pragma: no cover
```

Comment:
Darn, I delete this block in my last PR, since it had become just 3.7 everywhere, so you'll get a conflict here and you'll have to bring it back. Sorry about that!